web/packages/teleterm/src/ui/DocumentConnectMyComputer/DocumentConnectMyComputerSetup.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { useState } from 'react';
+import { Box, ButtonPrimary, Text } from 'design';
+
+import * as types from 'teleterm/ui/services/workspacesService';
+import Document from 'teleterm/ui/Document';
+
+interface DocumentConnectMyComputerSetupProps {
+  visible: boolean;
+  doc: types.DocumentConnectMyComputerSetup;
+}
+
+export function DocumentConnectMyComputerSetup(
+  props: DocumentConnectMyComputerSetupProps
+) {
+  const [step, setStep] =
+    useState<'information' | 'agent-setup'>('information');
```

Comment:
I'm not sure about using "step" as the word here since agent-setup is going to have its own set of steps, but I don't have a better idea and it's not that big of a deal.